endless-sky/.devcontainer/Dockerfile

```
@@ -20,7 +29,7 @@ RUN \
 # AppImage requirements
 RUN \
   apt-get update; \
-  apt-get install -y curl fuse libfuse2 rsync
+  apt-get install -y curl rsync
```

```suggestion
  apt-get install -y rsync
```
curl now installed on line 12